composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "778c2ba69a45162c2024867ab93f50bb",
+    "content-hash": "270c65f46ea48c6629d755ad2a1e8384",
```

Comment:
No idea why this hash was updated while the rest of the file didn't change.

Reply:
because of this: 
https://github.com/composer/composer/blob/1.5.2/src/Composer/Package/Locker.php#L65-L102
It depends of the composer.json content.